src/openms/include/OpenMS/ANALYSIS/ID/FragmentIndex.h

```diff
@@ -39,6 +39,41 @@ namespace OpenMS
       float precursor_mz;  // charge 1 precursor m/z
     };
 
+    struct sPeptide
+    {
+      UInt32 protein_idx_;
```

Comment:
adding doxygen comments like this would be of great help to others because it is such a central data structure.
e.g.
```suggestion
      UInt32 protein_idx_; //< The index in the protein database
```